### DISCUSSION TITLE
Additional field in the yaml for pre-curation contamination

### DISCUSSION THREAD

**Discussion by bistace** [ERGA-consortium/EARs#52] [Ideas]
Hello,

I was thinking that at the moment we only know how the contamination looks like post-curation and theoretically we should not see any (in the majority of cases). Do you think that it would be feasible/interesting to add a `blobplot_cont_png` field like it is done in the `Curated` assembly to show the contamination before curation to see the state of the genome after the assembly stage?

**Comment by diegomics** [comment]
This is a nice idea. But I would keep it optional. I will add the field in the stable version of the yaml & script. This version should be ready in the coming days. Thanks for the feedback, Benjamin :)

**Reply by bistace** [reply]
Thank you! Yes you are right, I think that keeping it optional would be the better choice here.

**Reply by diegomics** [reply]
I'm updating the YAML and makeEAR.py to the first stable version. I've been testing the addition of this optional field in the YAML and an optional section in the resulting PDF, but it's messing things up a little...
The rationale behind showing the post-curation contamination blob plot was to have proof that the curated assembly looks ok in that aspect. I think that, like for other things, if the reviewer (or the review requester) wants to add this kind of extra information to the reviewing process, the plot can be attached to the pull-request conversation, right?

**Reply by bistace** [reply]
Sorry for the late reply, I was out of the office. 

You are right that this can be attached to the pull-request, it was just to have a more convenient way to have a before/after comparison in the same file but don't think about it too much if this is messing things up or is too complicated!

**Comment by diegomics** [comment]
Following on this, I think that if we only maintain the post-curation blobplot, if that plot is showing something like a false-positive and that, it should be stated in the "Contamination notes" section that it was checked and resulted a false positive, despite what the plot is showing
